### PR TITLE
Release 0.32

### PR DESCRIPTION
**Pull request by AlanGriffiths**
We're still a couple of logind related PRs from release, but this gets the process rolling.

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,58 @@
...
+    - Bugs fixed:
+      . [Wayland] Implement subsurface support for touches. (Fixes #347)
+      . [Wayland] Implement Input regions. (Fixes #306)
+      . [Wayland] Implement/correct implementation of window state changes in
+        xdg-shell V6. (Fixes #297)
+      . [Wayland] Implement the common state changing logic in
+        WlAbstractMirWindow. (Fixes: #311)
+      . [Wayland] Don't hold a lock in mf::WlShmBuffer while potentially
+        destroying an object that uses it. (Fixes #355)
+      . [Wayland] Disconnect a sink when its target window is destroye
```
Comment (wmww) [inline debian/changelog]: destroye typo

**Comment by AlanGriffiths** [comment]
Almost there - we just need to activate the logind support. (Without breaking all the tests like #425)

**Comment by AlanGriffiths** [comment]
@Saviq we should be able to cherry-pick #426 and start the release process.

**Comment by AlanGriffiths** [comment]
Passed [test plan](https://wiki.ubuntu.com/Process/Merges/TestPlans/Mir) for 18.04/AMD64

**Comment by AlanGriffiths** [comment]
**FAILED** [test plan](https://wiki.ubuntu.com/Process/Merges/TestPlans/Mir) for 16.04: Running `miral-desktop` in a terminal window on 16.04 LTS starts the shell under X11 (Issue #428)

**Comment by AlanGriffiths** [comment]
Passed [test plan](https://wiki.ubuntu.com/Process/Merges/TestPlans/Mir) for 17.10/AMD64

**Comment by RAOF** [comment]
**FAILED** test plan for 18.10 - mesa-kms fails to initialise on my desktop (#435)

**Comment by AlanGriffiths** [comment]
Regression (seen on 18.04/gnome/wayland): `miral-shell --startup <apps>` not working (#437)

**Comment by AlanGriffiths** [comment]
@RAOF in the interests of making a release we've decided to ignore 18.10 for the moment.

We are aware that releasing to the Ubuntu archives is problematic, but we're good with updating ppa:mir-team/release (and using that for building snaps).

As and when we've fixed #435 (and anything else we want to pick) we can release 0.32.1 and upload to 18.10 and SRU to 18.04.

**Pull request merged**